diff -up awesome-3.5.1/awesomeConfig.cmake~ awesome-3.5.1/awesomeConfig.cmake
--- awesome-3.5.1/awesomeConfig.cmake~	2013-04-01 13:44:46.000000000 +0200
+++ awesome-3.5.1/awesomeConfig.cmake	2013-04-03 20:06:05.560633589 +0200
@@ -130,6 +130,7 @@ pkg_check_modules(AWESOME_COMMON_REQUIRE
 
 pkg_check_modules(AWESOME_REQUIRED REQUIRED
     glib-2.0
+    libev
     gdk-pixbuf-2.0
     cairo
     x11
