diff -up awesome-3.5.3/awesomeConfig.cmake.lua-52 awesome-3.5.3/awesomeConfig.cmake
--- awesome-3.5.3/awesomeConfig.cmake.lua-52	2014-03-29 18:17:20.000000000 +0100
+++ awesome-3.5.3/awesomeConfig.cmake	2014-04-02 18:14:57.232099602 +0200
@@ -64,8 +64,8 @@ a_find_program(CONVERT_EXECUTABLE conver
 include(FindDoxygen)
 # pkg-config
 include(FindPkgConfig)
-# lua 5.1
-include(FindLua51)
+# lua 5.2
+include(FindLua52)
 # }}}
 
 # {{{ Check if documentation can be build
@@ -189,7 +189,7 @@ else()
 endif()
 
 # Error check
-if(NOT LUA51_FOUND AND NOT LUA50_FOUND) # This is a workaround to a cmake bug
+if(NOT LUA52_FOUND AND NOT LUA51_FOUND AND NOT LUA50_FOUND) # This is a workaround to a cmake bug
     message(FATAL_ERROR "lua library not found")
 endif()
 
