From b0d4f4eb0e375b61b347a9ba1a0a99a459fad9ba Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Sun, 7 Sep 2014 19:33:18 +0200
Subject: [PATCH] Use version agnostic FindLua from cmake-3.0.0

---
 CMakeLists.txt      | 2 +-
 awesomeConfig.cmake | 9 ++-------
 2 files changed, 3 insertions(+), 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index fc7b84c..f8b518b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.0)
+cmake_minimum_required(VERSION 3.0.0)
 set(CMAKE_ALLOW_LOOSE_LOOP_CONSTRUCTS  TRUE)
 
 if(COMMAND cmake_policy)
diff --git a/awesomeConfig.cmake b/awesomeConfig.cmake
index 620c2b4..d95d9d3 100644
--- a/awesomeConfig.cmake
+++ b/awesomeConfig.cmake
@@ -60,8 +60,8 @@ a_find_program(CONVERT_EXECUTABLE convert TRUE)
 include(FindDoxygen)
 # pkg-config
 include(FindPkgConfig)
-# lua 5.1
-include(FindLua51)
+# lua
+include(FindLua)
 # }}}
 
 # {{{ Check if documentation can be build
@@ -189,11 +189,6 @@ else()
     message(STATUS "checking for __builtin_clz -- no")
 endif()
 
-# Error check
-if(NOT LUA51_FOUND AND NOT LUA50_FOUND) # This is a workaround to a cmake bug
-    message(FATAL_ERROR "lua library not found")
-endif()
-
 set(AWESOME_REQUIRED_LDFLAGS
     ${AWESOME_COMMON_REQUIRED_LDFLAGS}
     ${AWESOME_REQUIRED_LDFLAGS}
-- 
2.5.5

